nucypher/characters/base.py

```diff
@@ -116,7 +116,9 @@ def __init__(self,
 
         #
         # Operating Mode
-
+        if hasattr(self, '_interface_class'):  # TODO: have argument about meaning of 'lawful'
+            #                                         and whether maybe only Lawful characters have an interface
+            self.interface = self._interface_class(character=self)
```

Comment:
This is only needed when using a controller, consider moving this out of init to prevent interface creation for characters that use just the plain python API.